tests/e2e-playwright/tests/sleepers.py

```diff
@@ -0,0 +1,125 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+# pylint:disable=protected-access
+# pylint:disable=no-value-for-parameter
+# pylint:disable=too-many-arguments
+# pylint:disable=too-many-statements
+
+
+import re
+from collections.abc import Callable
+from typing import Final
+
+from playwright.sync_api import Page, WebSocket
+from pytest_simcore.playwright_utils import (
+    MINUTE,
+    SocketIOEvent,
+    SocketIOProjectStateUpdatedWaiter,
+    decode_socketio_42_message,
+)
+
+_NUM_SLEEPERS = 12
+_WAITING_FOR_CLUSTER_MAX_WAITING_TIME: Final[int] = 5 * MINUTE
+_WAITING_FOR_STARTED_MAX_WAITING_TIME: Final[int] = 5 * MINUTE
+_WAITING_FOR_SUCCESS_MAX_WAITING_TIME: Final[int] = _NUM_SLEEPERS * 1 * MINUTE
+
+
+def test_sleepers(
+    page: Page,
+    log_in_and_out: WebSocket,
+    create_new_project_and_delete: Callable[..., None],
+    start_and_stop_pipeline: Callable[..., SocketIOEvent],
+    product_billable: bool,
+):
+    # open service tab and filter for sleeper
+    page.get_by_test_id("servicesTabBtn").click()
+    _textbox = page.get_by_test_id("searchBarFilter-textField-service")
+    _textbox.fill("sleeper")
+    _textbox.press("Enter")
```

Comment:
Out of curiosity: How much of this is autogenerated by playwright? What I took away from @matusdrobuliak66's presentation was that you should basically be able to generate this kind of test by simply clicking through it in the ui, right? But when reading this test it appears quite man-made 😄.

Reply:
in the end I guess about 5% is auto-generated. I re-used @matusdrobuliak66 initial test and also played with the autogen but once you start refining the test, the autogeneration is not very useful anymore. I'd say it's cool for an initial trial and to see how it works.

Reply:
Hmm ok. Thanks for the clarification.